frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -195,7 +205,35 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
               loaded={pvcResourceLoaded}
               loadError={pvcResourceLoadError}
               pvcResource={pvcResource}
+              availableAccessModes={volumeSnapshotAnnotations?.[
+                snapshotPVCAccessModeAnnotation
+              ]?.split(',')}
             />
+            <FormGroup
+              fieldId="pvc-volume-mode"
+              className="co-restore-pvc-modal__input"
+              label="Volume Mode"
+              isRequired
+            >
+              <RadioInput
+                value="Filesystem"
+                key="Filesystem"
+                title="Filesystem"
+                onChange={handleVolumeMode}
+                inline
+                checked={volumeMode === 'Filesystem'}
```

Comment:
There must be some constant for modes and if not , we should create one IMO

Reply:
https://github.com/openshift/console/pull/8136#discussion_r576032168

Reply:
done